readthedocs/search/documents.py

```diff
@@ -48,6 +48,15 @@ class ProjectDocument(RTDDocTypeMixin, Document):
 
     modified_model_field = 'modified_date'
 
+    def get_queryset(self):
+        """
+        Don't include delisted projects.
+        This will also break in-doc search for these projects,
+        but it's not a priority to find a solution for this as long as "delisted" projects are
+        understood to be projects with a negative reason for being delisted.
```

Comment:
@ericholscher I wouldn't go very far to get in-doc search working for these projects. They're being unlisted for good reasons. But if we change the path to unlisted so there are other types of unlisted projects that should still be indexed, then we can revisit this?

Reply:
Yea, I think in-doc search should still work from the tool probably? But mostly just shows it might not be the right implementation of the idea, perhaps.

Reply:
I think the implementation is right as long as it's not causing any issues. If there are any real issues with this (which I find unlikely for the current usage), we can change it by implementing a feature whereby delisted projects can be indexed but excluded from global search results?

Reply:
Decision coming up about whether in-doc search + delisting is too complex in https://github.com/readthedocs/readthedocs-ext/issues/461#issuecomment-1444063694